test/controllers/api/episodes_controller_test.rb

```diff
@@ -81,7 +81,7 @@
     let(:podcast) { create(:podcast, prx_account_uri: "/api/v1/accounts/#{account_id}") }
     let(:episode_redirect) { create(:episode, podcast: podcast, published_at: nil) }
     let(:episode_update) { create(:episode, podcast: podcast, published_at: nil) }
-    let(:token) { StubToken.new(account_id, ['member']) }
+    let(:token) { StubToken.new(account_id, ['member feeder:read-private feeder:podcast-edit feeder:podcast-create feeder:episode feeder:episode-draft']) }
```

Comment:
Why leave `member` in here?

Reply:
Actually, I do see `member` and `admin` scopes in my JWT right now.  Are those for backwards compatibility or something?

Reply:
Those are in tokens right now for compatibility with the apps as currently deployed (until this patch is released, and its partners in the other repos) - at some point in the future, we can remove the meta-scopes.